_chapters/chapter_02.md

```diff
@@ -146,15 +163,23 @@ layout: rails_tutorial
 {% endhighlight %}
 {% endsteps %}
 
+{% aside %}
+### Classes in Ruby 
+
+The Ruby in 100 minutes tutorial discussed classes only briefly. All you really need to know is that they are: 
+
+> Abstract descriptions of a category or type of thing. They define what attributes and methods all objects of that type have.
+{% endaside %}
```

Comment:
I totally agree - I think this is okay for now but if we have time or it keeps bugging us we can adjust it.